autoload/lsp/ui/vim/documentation.vim

```diff
@@ -6,7 +6,7 @@ let s:last_popup_id = -1
 function! s:complete_done() abort
```

Comment:
It should be `complete_changed`.

Reply:
I didn't change it!
It was `s:complete_done()` before the PR, and I don't want to change it unless @prabirshrestha says so....

Reply:
@prabirshrestha ?

Reply:
It is a trivial thing but I think `complete_changed` is better name.
See https://github.com/prabirshrestha/vim-lsp/pull/921/files#diff-b569a8722a789fc65d0920e66f13bfb4282225eb0b8260e254b72f073cca4828R149

Reply:
OK. Renamed.